spec/integration/group_show_spec.rb

```diff
@@ -3,11 +3,11 @@
 RSpec.describe 'Group Index', type: :feature do
   describe 'Group' do
     before(:each) do
-      @user1 = User.create! name: 'Gordon', password: 'godanah', email: 'gbuldie@gmail.com'
+      @user1 = User.create! name: 'samara', password: 'samu@2022', email: 'samara@gmail.com'
       @group = @user1.groups.create! name: 'HBO', icon: 'icon'
       visit new_user_session_path
-      fill_in('Email', with: 'gbuldie@gmail.com')
-      fill_in('Password', with: 'godanah')
+      fill_in('Email', with: 'samara@gmail.com')
+      fill_in('Password', with: 'samu@2022')
```

Comment:
- [ ] I know you have tried working on the tests but currently six of them do not pass. This is the error I get when I run the command for the test

![image](https://user-images.githubusercontent.com/77106962/164949112-b2fcfab4-b6f1-43ff-8a72-20fc7d88f5c1.png)

This is most likely from the way you are linking the buttons. Kindly fix this. 👍